proto/tendermint/types/types.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
 package tendermint.types;
 
-option go_package = "github.com/tendermint/tendermint/types";
+option go_package = "github.com/tendermint/tendermint/proto/tendermint/types";
```

Comment:
@williambanfield, I see that this `go_package` configuration was introduced in #8065 - is there a specific reason for it? It breaks buf's proto generation if the package isn't configured to be the same as the rest of these files.